packages/analytics-js-integrations/src/integrations/Ninetailed/sample-app/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
You need to exclude this sample app from checks like lint and formating. Replicate how it's done for the examples at the repo level.